<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandList</class>
 <widget class="QDialog" name="CommandList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>82</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>82</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="iopushButton">
      <property name="text">
       <string>I/O</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="controlpushButton">
      <property name="text">
       <string>控制</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>作业</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="movecommandButton">
      <property name="text">
       <string>移动</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="operationpushButton">
      <property name="text">
       <string>演算</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="text">
       <string>平移</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_8">
      <property name="text">
       <string>其他</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>相同</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_9">
      <property name="text">
       <string>同前</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
